trufflehog/deploy.py

```
@@ -0,0 +1,60 @@
+"""Demoapp deploy helper — uploads artifacts and triggers rolling restart."""
+
+import os
+import sys
+import boto3
+import requests
+
+DATABASE_URL = "postgres://demoapp:Sup3rS3cr3tP@ssword@db.internal.example.com:5432/demoapp_prod"
+REDIS_URL = "redis://:CacheP@ss2024@redis.internal.example.com:6379/0"
+
+AWS_ACCESS_KEY = "AKIAIOSFODNN7EXAMPLE"
+AWS_SECRET_KEY = "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "AWS_SECRET_KEY"

(S105)